app/controllers/companies_controller.rb

```diff
@@ -44,6 +45,18 @@ def company_params
     params.require(:company).permit(:name, :description, :email, :phone)
   end
 
+  def company_registration_params
+    params.require(:company_registrations_form).permit(:name,
```

Comment:
Якось так можна:
```suggestion
    params
      .require(:company_registrations_form)
      .permit(:name, :description, :company_email,
                    :phone, :first_name, :last_name,
                    :user_email, :password, :password_confirmation)
```

І не треба збільшувати ліміт на кількість рядків.